core/download.ts

```diff
@@ -4,6 +4,7 @@ export interface TextResponse {
   readonly headers: Headers
   readonly ok: boolean
   parse(): Document | XMLDocument
+  parseJson(): null | object
```

Comment:
I just remember, that JSON can be also an array. What do you think if we will use `unknown` here?

Reply:
Definitely better decision, you are right, json can be almost any type in current app data